src/utils/makeReactNativeConfig.js

```diff
@@ -59,7 +59,6 @@ const getDefaultConfig = ({
   const platformProgressBar = haulProgressBar(platform);
   return {
     context: root,
-    devtool: bundle ? 'source-map' : 'eval-source-map',
```

Comment:
Have you verified that this is not needed?
- Chrome debugger should show sourcemaps
- When bundling, a sourcemap file should be generated

Reply:
Oh, should be? I see. OK, let me check this once more.

Reply:
It's OK now @satya164

Reply:
Does chrome devtools find and use those source maps? `eval-source-map` is there deliberately, otherwise we had problem with getting SM to work.

Reply:
Yeah, It's OK. Anyway, I think there is no reason for use `devtool` when you use `SourceMapDevToolPlugin`